model-metadata/CU-ensemble.yml

```
@@ -1,35 +1,40 @@
 team_name: "Columbia University"
 team_abbr: "CU"
-model_name: "Ensemble"
-model_abbr: "ensemble"
+model_name: "Columbia University Ensemble"
+model_abbr: "CU-ensemble"
 model_contributors: [
   {
     "name": "Rami Yaari",
     "affiliation": "Columbia University",
-    "email": "ry2460@cumc.columbia.edu"
+    "email": "ry2460@cumc.columbia.edu",
+    "orcid": "0000-0002-8808-8937"
   },
   {
     "name": "Teresa Yamana",
     "affiliation": "Columbia University",
-    "email": "tky2104@cumc.columbia.edu"
+    "email": "tky2104@cumc.columbia.edu",
+    "orcid": "0000-0001-8349-3151"
   },
   {
     "name": "Sen Pei",
     "affiliation": "Columbia University",
-    "email": "sp3449@cumc.columbia.edu"
+    "email": "sp3449@cumc.columbia.edu",
+    "orcid": "0000-0002-7072-2995"
   },
   {
     "name": "Jeffrey Shaman",
     "affiliation": "Columbia University",
-    "email": "jls106@cumc.columbia.edu"
+    "email": "jls106@cumc.columbia.edu",
+    "orcid": "0000-0002-7216-7809"
   }
 ]
 website_url: "https://blogs.cuit.columbia.edu/jls106/"
-license: "CC-BY-4.0"
-team_funding: "US NIH grant AI163023 and CDC 75D30122C14289"
-designated_model: true
-data_inputs: "State and national-level daily confirmed influenza hospital admissions, queried using covidcast R package. State and national-level ILINet surveillance data, queried using cdcfluview R package."
-methods: "An inverse-WIS weighted ensemble of several component models - an SEIRS compartmental model with EAKF, an ARIMA model, a random walk with drift, and the N-HiTS and N-BEATS deep-learning models."
-methods_long: "The dynamical model simulates influenza transmission in each state and the US using a humidity-driven SEIRS dynamics. Model variables and parameters are sequentially updated each week using the ensemble adjustment Kalman filter and new observations. Forecasts are generated by integrating the optimized model into the future. Autoregressive Integrated Moving Average model and baseline models use implementations available in the fable R package (ARIMA and RW, respectively).  We employ multivariate versions of N-HiTS and N-BEATS models as implemented in the darts python package, trained on modified state-level ILI data and hospitalization data. To build ensemble, the quantile distributions of the component models are weighted by the sum of inverse-WIS scores, over last 4 weeks. The 4-week window is target and location-specific and are recomputed at each forecast week."
+repo_url: "https://github.com/ramiyaari/Flusight-CU-Ensemble"
+license: "cc-by-4.0"
```

This should fix the model error

```suggestion
license: "CC-BY-4.0"
```